helix-lock/pom.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.apache.helix</groupId>
+    <artifactId>helix</artifactId>
+    <version>0.9.2-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>helix-lock</artifactId>
+  <packaging>bundle</packaging>
+  <name>Apache Helix :: Distributed Lock</name>
+
+  <properties>
+    <osgi.import>
+      org.apache.helix*,
+      org.slf4j*;version="[1.6,2)",
+      *
+    </osgi.import>
+    <osgi.export>org.apache.helix.lock*;version="${project.version};-noimport:=true</osgi.export>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
+      <version>1.17</version>
+    </dependency>
+    <dependency>
```

Comment:
These dependencies are the same in the parent pom. I think it would be more clear and easier to maintain if you remove these dependencies in this child pom, because this child pom file will inherit all properties and dependencies from parent POM. Unless you want to use a different version/additional dependency.
Same with following duplicate dependencies.

Reply:
I actually don't find this one in parent pom. Do you have another example that is in the parent?

Reply:
Oh, it is not `snakeyaml`. I actually selected a block but github doesn't show it. I meant this part as an example: https://github.com/apache/helix/blob/d6ba1f3e792651e250d60b3df1fc6b537934f69e/pom.xml#L404-L413
Do you think the dependencies could be removed?

Reply:
yeah, removed.